java-manta-client-unshaded/src/main/java/com/joyent/manta/config/BaseChainedConfigContext.java

```diff
@@ -446,78 +443,133 @@ public void overwriteWithContext(final ConfigContext context) {
 
             this.privateKeyContent = context.getPrivateKeyContent();
         }
+
         if (isPresent(context.getPassword())) {
             this.password = context.getPassword();
         }
+
         if (isPresent(context.getHttpsProtocols())) {
             this.httpsProtocols = context.getHttpsProtocols();
         }
+
         if (isPresent(context.getHttpsCipherSuites())) {
             this.httpsCipherSuites = context.getHttpsCipherSuites();
         }
+
         if (context.noAuth() != null) {
             this.noAuth = context.noAuth();
         }
+
         if (context.disableNativeSignatures() != null) {
             this.disableNativeSignatures = context.disableNativeSignatures();
         }
+
         if (context.getHttpBufferSize() != null) {
             this.httpBufferSize = context.getHttpBufferSize();
         }
-        if (context.getTcpSocketTimeout() != null) {
-            this.tcpSocketTimeout = context.getTcpSocketTimeout();
-        }
-        if (context.getConnectionRequestTimeout() != null) {
-            this.connectionRequestTimeout = context.getConnectionRequestTimeout();
-        }
-        if (context.getExpectContinueTimeout() != null) {
-            this.expectContinueTimeout = context.getExpectContinueTimeout();
-        }
+
+        overwriteWithContextTimeouts(context);
+
         if (context.verifyUploads() != null) {
             this.verifyUploads = context.verifyUploads();
         }
+
```

Comment:
All these whitespaces have been added to the original file for whitespace consistency.